oauth-common/src/main/java/io/strimzi/kafka/oauth/common/Config.java

```diff
@@ -128,7 +131,7 @@ public boolean getValueAsBoolean(String key, boolean fallback) {
         }
     }
 
-    private boolean isTrue(String result) {
+    public static boolean isTrue(String result) {
```

Comment:
of course this was already here but why we are supporting all these possible values and not just "true/false"?

Reply:
The only reason would be usability / user experience. Some people may in some situation prefer 'y' over 'true', or '0' over 'false'. It will be a string in input in any case, and needs to be converted to boolean somehow. This code here makes it both versatile, and specific - as we can always point to the algorithm with a [url](https://github.com/strimzi/strimzi-kafka-oauth/blob/master/oauth-common/src/main/java/io/strimzi/kafka/oauth/common/Config.java#L131-L140).

@ppatierno Do you see some negatives of this choice?

Reply:
in general, I don't like to give too much freedom to users :-)
Anyway I can live with it.